src/pantry.js

```diff
@@ -1,7 +1,97 @@
 class Pantry {
   constructor(userIngredients) {
     this.contents = userIngredients;
+    this.filtered
+    this.goodAmount;
+    this.notEnough = []
+  }
+
+  determineIfUserHasEnoughIngredients(recipe) {
+    let AllIngredientInPantryIds = this.contents.map(ing =>{
+      return ing.ingredient 
+    })
+
+     this.filtered = recipe.ingredients.filter(ingredient => {
+      return AllIngredientInPantryIds.includes(ingredient.id)
+    })
+
+     this.goodAmount =  this.filtered.filter(ing =>{
+      if (AllIngredientInPantryIds.indexOf(ing.id) > -1) {
+        if (!this.contents[AllIngredientInPantryIds.indexOf(ing.id)].amount >= ing.quantity.amount) {
```

Comment:
maybe we should be assigning a variable to this value since we are using it more than once in this method, just an idea: something like:

let specificPantryAmount = `this.contents[AllIngredientInPantryIds.indexOf(ing.id)].amount`

Reply:
Agreed, it gets kinda messy here